src/components/PostComment.jsx

```diff
@@ -0,0 +1,68 @@
+import { useContext, useState } from "react"
+import { postCommentByArticleId } from "../api"
+import { UserContext } from "../contexts/userContext"
+
+const PostComment = ({article_id, setComments}) => {
+    const [postComment, setPostComment] = useState("")
+    const [isError, setIsError] = useState(false)
+    const [isLoading, setIsLoading] = useState(false)
+    const {user} = useContext(UserContext)
+
+    const handleCommentInput = (event) => {
+        setPostComment(event.target.value)
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        
+        const newCommentInput = {
+            author: user.username,
+            body: postComment,
+            created_at: new Date().toISOString(),
+            votes: 0,
+        }
+
+        setIsLoading(true)
+        setTimeout(() => {
```

Comment:
I don't think this first Timeout is necessary, it only seems to be creating a false delay?